src/app/screens/connectors/ConnectGaloy/index.tsx

```diff
@@ -266,12 +279,14 @@ export default function ConnectGaloy(props: Props) {
           navigate("/test-connection");
         }
       } else {
-        toast.error(`Connection failed (${validation.error})`);
+        toast.error(
```

Comment:
Should we align all connectors to use `ConnectionErrorToast`?  
Asking because I saw you switched it in the other PR as well.